sdk/core/azure-core-tracing-opentelemetry/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 1.0.0-beta.4 (Unreleased)
+- Keyvault: add az namespace info attribute to all outgoing spans.
```

Comment:
Is this something good to mention in KV changelogs too ?

Reply:
This would rather be a value add feature to tracing than to KV so we can skip KV changelogs for this one.